packages/cli/package.json

```diff
@@ -132,6 +133,9 @@
     "tar": "^6.1.12",
     "terser-webpack-plugin": "^5.1.3",
     "util": "^0.12.3",
+    "vite": "^4.4.9",
+    "vite-plugin-html": "^3.2.0",
+    "vite-plugin-node-polyfills": "^0.14.1",
```

Comment:
Had a thought, perhaps we should make some of these peer deps while it's still in this phase? At least the large ones. `vite-plugin-html` and `@vitejs/plugin-react` are both slim, but `vite` is over 3 MB and brings in `esbuild`, and somehow `vite-plugin-node-polyfills` is over 13 MB. Apparently it's all examples shipped in the package itself, opened https://github.com/davidmyersdev/vite-plugin-node-polyfills/pull/50. Ideally that would get merged and published soonish and we could just leave `vite` as a peer dep?